include/roboteam_robothub/RobotHub.h

```diff
@@ -6,35 +6,34 @@
 #define RTT_ROBOTHUB_H
 
 #include <libusb-1.0/libusb.h>
-
 #include <networking/Publisher.h>
 #include <networking/Subscriber.h>
 #include <roboteam_proto/AICommand.pb.h>
-#include <roboteam_proto/Setting.pb.h>
 #include <roboteam_proto/RobotFeedback.pb.h>
-#include "roboteam_proto/State.pb.h"
+#include <roboteam_proto/Setting.pb.h>
 
 #include "GRSim.h"
-#include "utilities.h"
 #include "constants.h"
+#include "roboteam_proto/State.pb.h"
```

Comment:
Looking at the other changes, I think this line should be moved up, and " " be replaced with < >